mw-webapp/src/component/logotype/Logotype.tsx

```diff
@@ -0,0 +1,43 @@
+import clsx from "clsx";
+import logo from "src/assets/mastersWayLogo.svg";
+import logoLight from "src/assets/mastersWayLogoLight.svg";
+import logoPurple from "src/assets/mastersWayLogoPurple.svg";
+import {Image} from "src/component/image/Image";
+import {useGlobalContext} from "src/GlobalContext";
+import {Theme} from "src/utils/ThemeWorker";
+import styles from "src/component/verticalContainer/VerticalContainer.module.scss";
+
+export const LOGO_TEXT = "Master's way";
+
+/**
+ * Logotype props
+ */
+interface LogotypeProps {
+
+  /**
+   * Additional custom class name
+   */
+  className?: string;
+
+  /**
+   * IsLogotype has another secondary color
+   */
+  isColorSecondary?: boolean;
+}
+
+/**
+ * Logotype
+ */
+export const Logotype = (props: LogotypeProps) => {
```

Comment:
1. let's make component ThemedImage instead. It could be used not only with logo
2. components should not be depend on global context

interface:
sources: Map<Theme, string-src>
theme: theme

Reply:
created ThemedImage component